### ISSUE TITLE
[BUG] `Plot count: Unknown` when running remote Harvesters

### ISSUE THREAD

**Issue by cayolblake** [Chia-Network/chia-blockchain#5881]
**To Reproduce**

1. Follow official docs for running Harvesters remotely.
2. Made sure all Harvesters are connected successfully to farmer.
3. On full-node/farmer/wallet machine, run, `chia farm summary`
4. You get the following output.

```
Connection error. Check if harvester is running at 8560
Farming status: Farming
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: Unknown
Total size of plots: Unknown
Estimated network space: 9861.239 PiB
Expected time to win: Unknown
Note: log into your key using 'chia wallet show' to see rewards for each key
```

**Expected behavior**
```
Plot count: 133
Total size of plots: 13.165054438842162 TiB
```

- Interface: `CLI`
- OS: `Linux`
- OS Version/Flavor: `Fedora`
- chia-blockchain Version: `1.1.6`

**Comment by baizuchong-net** [comment]
你要清楚的是，配集群是一主多从，只有主需要 链接和配置，从只需要ca和start Harvester

**Comment by jzmexec** [comment]
My main machine run "chia farm summary" just get the plots count as main machine ,Not the sum in my harvester network. Can you give me some idea?

**Comment by jzmexec** [comment]
@baizuchong-net @richardkiss @baizuchong-net

**Comment by guojiazhuxi** [comment]
So do I

**Comment by cayolblake** [comment]
And update from any of the developers? 🤔 

@arvidn @hoffmang9

**Comment by kiss81** [comment]
As we are supposed to run multiple harvesters and one main farm it would be nice if the main farm could give feedback. I think this should fit perfectly  in the farm summary. Also the GUI is acting very strange. If you're not farming a plot locally it only shows your external harvester is connected but you don't  see any other feedback. Kind of annoying.

**Comment by cayolblake** [comment]
And update from any of the developers? 🤔 

@arvidn
@hoffmang9

**Comment by kiss81** [comment]
Not yet fixed in 1.17 I am running external harvesters only now, but in doubt if it works correctly because of this bug. I see plots passing the filter on my harvesters so I hope all is ok. It would be nice to have a confirmation of this on my farmer as well. Please give this priority when possible  @arvidn @hoffmang9 Thank you!

**Comment by cayolblake** [comment]
@kiss81 How you `verify` that your plots passing the filter?

**Comment by kiss81** [comment]
> @kiss81 How you `verify` that your plots passing the filter?

Set your loggin level to INFO at your  harvester config.yaml

**Comment by cayolblake** [comment]
@kiss81 It is already set to INFO from day one. What do you look for to verify?

**Comment by kiss81** [comment]
> And update from any of the developers? thinking
> 
> @arvidn @hoffmang9



> @kiss81 It is already set to INFO from day one. What do you look for to verify?

cat .chia/mainnet/log/debug.log |grep proof

Should give you lines like this: 
2021-06-06T22:47:27.257 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming a2585bd6f7... Found 0 proofs. Time: 0.45515 s. 

Still I am not sure if it's really working if it shows these lines as it should as I didn't find anything yet.

**Comment by cayolblake** [comment]
Yes, I have lots of those since day one as well, yet, I believe a confirmation form someone whom won any XCH would confirm this. Did you win any XCH yet?

**Comment by kiss81** [comment]
I did not find a block using this external harvester method. Would like to know this at well. I think it would give some confidence if the main farm wallet would show something about farming (and the external harvesters), so I really think this is a bug / missing feature.

**Comment by cayolblake** [comment]
Not the first one to hear that from.

Developers seems kind of super busy.

**Comment by kiss81** [comment]
I understand there are lot's of other issues too, but this is one of the core functionalities that is not working / reporting properly.

**Comment by kiss81** [comment]
I think it's being fixed soon. Fingers crossed:
https://github.com/Chia-Network/chia-blockchain/pull/6527

**Comment by cayolblake** [comment]
`hopefully` it gets merged.

**Comment by kiss81** [comment]
> All hands are on pooling. Your PRs @cayolblake are welcome.

Thanks for replying.  can understand pools are high priority at the moment. On the other hand there are quite some farmers that won't replot what they already have (like myself). The fix for this issue will make it easier to find issues (if there are any) on my system.

**Comment by cayolblake** [comment]
@hoffmang9 I'm really interested in providing support and PRs if possible, especially that the core lib is in Cpp and upper layer in Python (if I'm getting this right), however, the way Open Source projects work is of high attendance of the core dev team and answering questions/inquiries/clarifications and rapidly and interactively enhancing the high-level documentation and the inline code documentation wherever needed.

Unfortunately, none of these points exists so far in Chia. The recommended app for communication `keybase`, I dunno how to put it otherwise but, simply, it's horrible. Not sure why on earth didn't you go for something like Slack, which is orders of magnitude more stable, faster and user friendly. Also, its like a super crowded coffee shop where the owners are so tired of the noise they left.

I'm really sorry to be the one stating this, but for the sake of success of this quite interesting project, and for people not just looking to make a buck (tho all respect to those as well), it might be a little annoying.

Many many things needs adjustment and proper planning in such guru projects with global traction. I've seen many projects, products, even companies, fail because of their overwhelming success they left the little things unmanaged and unhandled that it ate them up alive. Literally by the word!

That's my two cents on this. @hoffmang9 I believe someone already started providing some PR in #6527 tho it needs manual validation and approval since he/she is a new contributor, which sound like never gonna happen since all team's hand are on pooling, which in my humble opinion, a reaction to the outraging communities, not a planned one, and if this was planned in any way, from a software development project management perspective, none of the issues/PRs would've been left untouched.

Again, this is out of respect to the core idea and its value and out of love, nothing else. 

All Chia team's efforts are appreciated after all.

**Comment by hoffmang9** [comment]
The reaction to focus on pooling is far less about the outrage and far more about the fact that it requires a replot. Every day that goes by is a day lost to not wasting people's time and resources around re-plotting even more space. That's not a fun answer for those of you who want support or other features but it is the right and very hard call right now.

We've doubled the team in the last 30 days but mythical man months being what they are - you're not going to feel that for at least a few more weeks. It also doesn't help that as we're hiring the very best - they have to give notice and take some time to be replaced - as they were usually critical to the functioning of their prior role. Those folks will be appearing soon.

Priorities always suck when what you want isn't prioritized - I get it. But we'd rather make hard decisions and stick with them to get to the best product possible.

**Comment by eFishCent** [comment]
> @hoffmang9 I'm really interested in providing support and PRs if possible, especially that the core lib is in Cpp and upper layer in Python (if I'm getting this right), however, the way Open Source projects work is of high attendance of the core dev team and answering questions/inquiries/clarifications and rapidly and interactively enhancing the high-level documentation and the inline code documentation wherever needed.
> 

I have done PRs without any of those requirements you listed. The code is pretty cleanly written. If you read through it, you should be able to add features pretty easily, especially the python code. Not sure why you're claiming it's so challenging 🤷

**Comment by cayolblake** [comment]
@hoffmang9 Can't say I don't agree with you. Your reply makes lots of sense on many angles.

For the problem of the overhead/concern of replotting, I guess this problem kind of resolved itself, luckily, with the release of multithreaded plotter projects reducing time by %90 on basic specs without expensive hardware, so this kind of eases the friction and barrier to replot.

This was inevitable and would've preferably been the highest priority problem to tackle since day one and would've avoided many negative speakers and dystopian minds from libeling Chia all over the globe and governments taking aggressive actions toward crypto in general as a result of the economical disruptions resulting from the shortage of storage hardware besides the already ongoing silicon shortage crisis worldwide.

Not sure if these are the reasons why XCH value is on a free fall for the past days but they probably contributed to it.

It's very clear Chia hires elite calibers, but still there should be some dedication to the freely-contributing open source believers as well whom invest the time to report issues (not whine about it) and discuss/suggest/contribute solutions to it.

Many open source projects whom have strong financial backing hires one or two members just to keep surfing the community waters and at the very least organized what's going on with GitHub issue tracker, which btw needs a huge cleaning sprint removing all the non-reports complains and request for support. 

Thoughts about building a large KB or FAQ and keep updated daily with users common experiences?

**Comment by mariano54** [comment]
Fixed by 1.2.0
